protected/migrations/fix_import/m220218_062834_update_url_image_tab.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+class m220218_062834_update_url_image_tab extends CDbMigration
+{
+	public function safeUp()
+	{
+	    Yii::app()->db->createCommand("update image set url = replace(url, 'http://gigadb.org/','https://assets.gigadb-cdn.net/images/datasets/') where url like 'http://gigadb.org/%';")->execute();
+        // The empty value would be converted to null after set up the test database from csv files, so the following step is needed to convert it back to empty again.
```

Comment:
Hi @kencho51 

I dug a little deeper and figured out the reason of the undesirable behaviour:
it's explained here: https://handlebarsjs.com/guide/builtin-helpers.html#if

>You can use the if helper to conditionally render a block. If its argument returns false, undefined, null, "", 0, or [], Handlebars will not render the block.

the code where that ``if`` test is used is ``ops/configuration/yii-conf/migration.php.dist``, line 11.

You can see that in the existing dev test data ``data/dev/image.csv`` with id 147, is a record with "" in the url.

The resulting Yii migration file ``protected/migrations/data/dev/m200529_050010_insert_data_image_tab.php`` has this section:
```
        $this->insert('image', array(
            'id' => '147',
            'tag' => 'CS icon1',
            'license' => 'public domain',
            'photographer' => 'Shashaank Vattikuti ',
            'source' => 'Gigascience',
        ));
```

where there is no url key because the if test has resulted in the key/value not to be added since the if test considers ``""`` as false by default.

So the take away of this comment is:

 It's better to fix the csv to migration javascript code to allow empty string rather than adding that dirty workaround in the Yii migration script.

The documentation I linked above is very well aware that the ``if`` behaviour might be indesirable in some context, as they add this hint on how to fix it:

>**Sub-Expressions**
>
>Helpers are the proposed way to add custom logic to templates. You can write any helper and use it in a sub-expression.
>
>For example, in checking for initialization of a variable the built-in #if check might not be appropriate as it returns false for empty collections (see [Utils.isEmpty](https://handlebarsjs.com/api-reference/utilities.html#handlebars-utils-isempty-value)).